network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/we_found_section.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<section class="tw-dark tw-my-7 large:tw-my-[120px]">
+  <div class="container tw-max-w-3xl tw-mx-auto">
+    <h4 class="tw-text-lg medium:tw-text-[1.375rem] tw-uppercase tw-font-sans tw-font-bold tw-mb-5">{% trans 'Ultimately, we found:' %}</h4>
+
+    <div class="tw-flex tw-flex-col tw-space-y-5">
+      <div class="tw-flex tw-flex-col medium:tw-flex-row tw-gap-3">
+        <span class="tw-text-4xl tw-text-white">{% trans '01' %}</span>
+        <p class="tw-font-changa tw-text-3xl medium:tw-text-[2.375rem] tw-mb-0">{% blocktrans trimmed %}People overwhelmingly feel that using YouTube’s user controls <span class="tw-text-[#FF3D3D]">does not change</span> their recommendations at all.{% endblocktrans %}</p>
+      </div>
+
+      {# Add special lowercase aria label to stop screen readers from reading as acronym #}
+      <span class="tw-uppercase tw-text-lg medium:tw-text-[1.375rem] tw-text-white tw-font-bold medium:tw-ml-[60px]" aria-label="{% trans 'and' %}">{% trans 'and' %}</span>
+
+      <div class="tw-flex tw-flex-col medium:tw-flex-row tw-gap-3">
+        <span class="tw-text-4xl tw-text-white">{% trans '02' %}</span>
+        <p class="tw-font-changa tw-text-3xl medium:tw-text-[2.375rem] tw-mb-0">{% blocktrans trimmed %}They’re right. YouTube’s user control mechanisms <span class="tw-text-[#FF3D3D]">don’t effectively prevent</span> “unwanted” recommendations.{% endblocktrans %}</p>
```

Comment:
**Suggestion** Same as above (`strong` rather than `span`?).

Reply:
This actually doesn't require bold so i'll leave as is for now haha